tests/integration_tests/test_account.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from eth_account import Account
+
+from .utils import ADDRS, w3_wait_for_new_blocks
+
+
+def test_get_transaction_count(ethermint):
```

Comment:
Can you please add a test that compares the response to geth?

Reply:
Added, thanks.